frontend/src/metabase/query_builder/components/AggregationPopover/AggregationPopover.jsx

```diff
@@ -156,29 +157,24 @@ export default class AggregationPopover extends Component {
   }
 
   renderItemExtra(item, itemIndex) {
+    let content;
+
     if (item.aggregation?.description) {
+      content = item.aggregation.description;
+    } else if (item.metric) {
+      content = <QueryDefinitionTooltip type="metric" object={item.metric} />;
+    }
+
+    if (content) {
       return (
-        <div className="p1">
-          <Tooltip tooltip={item.aggregation.description}>
+        <AggregationMetricItemTooltip>
```

Comment:
This styled component could be replaced with Mantine's Box component from `metabase/ui`
```suggestion
        <Box ml="auto" p="0.5rem">
```